docs/installation/strategy.md

```diff
@@ -0,0 +1,15 @@
+# Installation Strategies
+
+While you are free to explore any deployments but the recommended stratgies are :
+
+| *OCP 4.13*      | *Microshift*     | *RHEL*  |  *ROSA* | *Kind* |
+| ------------- | -------------  | ----- | ----- | ----|
+| kepler-operator | Manifests  | RPM | Manifests | Helm Charts, Manifests, kepler-operator
```

Comment:
@SamYuan1990 can you verify this table? Is it possible to use helm charts with microshift?

Reply:
I suppose as we have microshift and kind  in our CI, we can ensure those works. But I have no idea with OCP, RHEL, ROSA...
I suppose helm works with microshift
https://github.com/sustainable-computing-io/kepler-helm-chart/actions/runs/5487590207/jobs/9999243718#step:4:567
https://github.com/sustainable-computing-io/kepler-helm-chart/actions/runs/5487590207/jobs/9999243718#step:8:7

@husky-parul , @rootfs please let me know if anything further I can help with.